src/cli/commands/lint.ts

```diff
@@ -137,8 +140,15 @@ async function lint(name: string, flags: any, command: Lint, rules?: RuleCollect
   if (flags.verbose) {
     command.log(`Linting ${name}`);
   }
-  const spec: IParserResult = await readParsable(name, flags.encoding);
-  const spectral = new Spectral();
+
+  let targetUri = name;
+  if (!startsWith(name, 'http')) {
```

Comment:
@P0lip looks like if name is `"httpModels/somefile.json" it will think it is a URL. Can we switch to`http(s)://` regex in the test PR you are working on? #264

Reply:
@philsturgeon sure, I'll make that happen.